### PR TITLE
Adapt to latest changes in OCP\Color

### PR DESCRIPTION
**Pull request by eneiluj**
refs #2705

The r,g,b attributes are now private.

### CODE REVIEW

**Comment by eneiluj** [comment]
I don't get Psalm complaints...

**Comment by juliushaertl** [comment]
Psalm uses the meta package to test against the OCP namespaces and this has not been updated yet: https://github.com/ChristophWurst/nextcloud_composer (should auto-resolve after the nightly bump)

File: lib/Service/SessionService.php [hunk 1]
```diff
@@ -228,6 +228,6 @@ private function getColorForGuestName(string $guestName = null): string {
 		$guestName = $this->userId ?? $guestName;
 		$uniqueGuestId = !empty($guestName) ? $guestName : $this->secureRandom->generate(12);
 		$color = $this->avatarManager->getGuestAvatar($uniqueGuestId)->avatarBackgroundColor($uniqueGuestId);
-		return sprintf("#%02x%02x%02x", $color->r, $color->g, $color->b);
+		return sprintf("#%02x%02x%02x", $color->red(), $color->green(), $color->blue());
```
Comment (juliushaertl) [inline lib/Service/SessionService.php:231]: As now available with the new API:

```suggestion
		return $color->name();
```

Comment (eneiluj) [reply lib/Service/SessionService.php:231]: Awesome!

**Comment by juliushaertl** [comment]
Retriggered static analysis since https://github.com/ChristophWurst/nextcloud_composer/commit/7367ca1960a63b8e0ec9c504f40fa4f743a4edd2 is there after the nightly auto update.

**Pull request merged**